### ISSUE TITLE
Support flags passing when generating packages

### ISSUE THREAD

**Issue by domenkozar** [input-output-hk/stack2nix#7]
Example: https://github.com/input-output-hk/cardano-sl/blob/master/stack.yaml#L5

Most probably depends on #5

**Comment by jmitchell** [comment]
We also need to support the `ghc-options` directive. Example: https://github.com/input-output-hk/cardano-sl/blob/423a1e5dfd87a693e2046dcabc767a001594d1eb/stack.yaml#L141

**Comment by MMesch** [comment]
another project that requires this feature:

https://github.com/AccelerateHS/accelerate-fft/blob/master/accelerate-fft.cabal

**Comment by cdepillabout** [comment]
Has this been fixed by #148?

**Comment by domenkozar** [comment]
Still need to support `ghc-options`, although that should be a new issue probably.
